lobby/src/main/java/org/triplea/lobby/server/db/MutedMacController.java

```diff
@@ -35,15 +34,11 @@ public MutedMacController(final Database database) {
    *
    * @throws IllegalStateException If an error occurs while adding, updating, or removing the mute.
    */
+  @Override
   public void addMutedMac(final User mutedUser, final @Nullable Instant muteTill, final User moderator) {
     checkNotNull(mutedUser);
     checkNotNull(moderator);
 
-    if (muteTill != null && muteTill.isBefore(now())) {
```

Comment:
checking upstream callers, we grab a time from user and add it to now, and we enforce the time value to be posistive. `mutetill` will never be in the past.

Reply:
Doesn't this break the unmute feature?